resolver/uuidResolver.go

```diff
@@ -0,0 +1,77 @@
+package resolver
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Financial-Times/uuid-utils-go"
+	"time"
+)
+
+const dateTimeFormat = "2006-01-02T15:04:05.000Z0700"
+
+type UuidsAndDateResolver interface {
+	Resolve(reqData []byte, respData []byte) (*UuidsAndDate, error)
+}
+
+type UuidsAndDate struct {
+	UuidArr      []string
+	LastModified string
+}
+
+type fromRequestResolver struct {
+}
+
+func NewUuidResolver() UuidsAndDateResolver {
+	return &fromRequestResolver{}
+}
+
+func (r *fromRequestResolver) Resolve(reqData []byte, respData []byte) (*UuidsAndDate, error) {
+	cc := &contentCollection{}
+	err := json.Unmarshal(reqData, cc)
+	if err != nil {
+		return nil, fmt.Errorf("Unmarshalling error: %v", err)
+	}
+
+	uuidArr, err := r.resolveUuids(cc)
+	if err != nil {
+		return nil, err
+	}
+
+	lastModified, err := r.resolveLastModified(cc)
+	if err != nil {
+		return nil, err
+	}
+
+	return &UuidsAndDate{uuidArr, lastModified}, nil
+}
+
+func (*fromRequestResolver) resolveUuids(cc *contentCollection) ([]string, error) {
```

Comment:
I can be, I don't want it to be. I like my functions attached to receivers :D

Still, if you say that this is "the go way", I'll do it